src/main/kotlin/com/team4099/robot2022/commands/shooter/AimAssist.kt

```diff
@@ -0,0 +1,39 @@
+package com.team4099.robot2022.commands.shooter
+
+import com.team4099.lib.units.base.inMeters
+import com.team4099.lib.units.base.meters
+import com.team4099.lib.units.derived.rotations
+import com.team4099.lib.units.inRadiansPerSecond
+import com.team4099.lib.units.perSecond
+import com.team4099.robot2022.commands.drivetrain.OpenLoopDriveCommand
+import com.team4099.robot2022.config.Constants.VisionConstants
+import com.team4099.robot2022.config.Constants.ShooterConstants
+import com.team4099.robot2022.subsystems.Drivetrain
+import com.team4099.robot2022.subsystems.Shooter
+import com.team4099.robot2022.subsystems.Vision
+import edu.wpi.first.wpilibj2.command.CommandBase
+import org.photonvision.targeting.PhotonTrackedTarget
+
+class AimAssist : CommandBase() {
```

Comment:
what exactly are we trying to do here? are we translating towards the target, or are we trying to rotate to point at it?

imo because we are pretty much only taking the fender shot the really useful aim assist is just translating left/right until the target is centered